src/styles/_common/_colors.scss

```diff
@@ -35,7 +35,9 @@ $palette: (
     'modal': #6b6b6b,
     'header': #545d69,
     'close': #b2b9c2,
-    'secondary': rgba(0, 60, 88, 0.7)
+    'secondary': rgba(0, 60, 88, 0.7),
+    'sorting-items': #788392,
+    'helper-copy': 788392
```

Comment:
@telmaantunes because we are using a context logic, some colors will be repeated.